dolphinscheduler-ui/src/views/datasource/list/use-form.ts

```diff
@@ -118,7 +124,23 @@ export function useForm(id?: number) {
             return new Error(t('datasource.jdbc_format_tips'))
           }
         }
+      },
+    testFlag: {
+      trigger: ['input'],
+      validator() {
+        if (state.detailForm.testFlag ===undefined) {
```

Comment:
The empty value is not only undefined. So  it is better to take the opposite here.